react/features/base/participants/middleware.js

```diff
@@ -185,6 +189,40 @@ StateListenerRegistry.register(
                 localParticipantIdChanged(LOCAL_PARTICIPANT_DEFAULT_ID));
     });
 
+/**
+ * Registers listeners for participant change events.
+ */
+StateListenerRegistry.register(
+    state => state['features/base/conference'].conference,
+    (conference, store) => {
+        if (conference) {
+            // We joined a conference
```

Comment:
We need to clear the raise hand indicator when the participant leaves the conference. Otherwise the indicator is displayed locally for that user when they join a conference, but their hand is not really raised.

Reply:
Good call, thanks! On a side note though, I think we should have a central place where we do these cleanups, not one flag after the other. That goes against that feature separation though, so not sure. Anyhow, I did it here, we'll see where it goes later.